go/oasis-test-runner/scenario/e2e/runtime/txsource.go

```diff
@@ -478,12 +476,6 @@ func (sc *txSourceImpl) Run(childEnv *env.Env) error {
 		return fmt.Errorf("WaitNodesRegistered: %w", err)
 	}
 
-	// Wait for second epoch, so that runtimes are up and running.
```

Comment:
I moved this step directly into the workloads that do runtime queries (`runtime` & `queries` workloads). A benefit for this is that other workloads now start sooner. 

Though, the main reason for moving this is so that `runtimes` workload starts doing queries sooner - what happened before (rarely) was that the runtime workload would submit the first runtime transaction more than `ProposerTimeout` (20) blocks after the 2nd epoch started, leading to nodes triggering proposer timeout on the first transaction received (since triggering a proposer timeout is faster operation than the proposer  proposing a batch and submitting a commitment). With this change hopefully that doesn't happen and we can keep the `AssertNoRoundFailures` check in for the txsource-short workload.

Am also considering adding a short delay before exectuor nodes trigger "proposer timeout", to avoid triggering a timeout always when there are no transactions for `ProposerTimeout` blocks.